Docs - Add beter docs for `permission` on `PanelGroup` / `ObjectList` etc (#10927)
wagtail/wagtail: docs/reference/pages/panels.md

Without a top-level panel definition, a `FieldPanel` will be constructed for each field in your model. If you intend to hide a field on the Wagtail page editor, define the field with [`editable=False`](django.db.models.Field.editable). If a field is not present in the panels definition, it will also be hidden.

(panels_permissions)=

### Permissions

Most panels can accept a `permission` kwarg, allowing the set of panels or specific panels to be restricted to a set permissions.
See [](permissions_overview) for details about working with permissions in Wagtail.

> In this example below, 'notes' will be visible to all editors, cost/details will only be visible to those with the 'submit' permissions, budget approval will be visible to super users only. Super users will have access to all fields.

Reviewer: nitpick:

```suggestion
In this example, 'notes' will be visible to all editors, 'cost' and 'details' will only be visible to those with the `submit` permission, 'budget approval' will be visible to super users only. Note that super users will have access to all fields.
```
